tests/ballerina-unit-test/src/test/resources/test-src/io/csv_io.bal

```diff
@@ -66,5 +66,8 @@ function getTable(string filePath, string encoding, io:Separator fieldSeperator)
         return total;
     } else if (tableResult is error) {
         return tableResult;
+    } else {
+        error e = error("Record channel not initialized properly");
```

Comment:
Should this go to error details?
ex:
error e = error(IO_ERROR_CODE, { message : "Record channel not initialized properly" });

Reply:
Fixed in 712af3e